utilities/authContext.tsx

```diff
@@ -0,0 +1,83 @@
+import { useRouter } from "expo-router";
+import { createContext, PropsWithChildren, useState } from "react";
+import { init, User } from "@instantdb/react-native";
+
+const APP_ID = "81185f3e-d773-4c94-a6cb-c08ebe7bdf22";
+const db = init({ appId: APP_ID });
```

Comment:
nit: I would move these out into a separate `db` file, because you'll probably want to use this to make queries later